### ISSUE TITLE
Integration with org-capture

### ISSUE THREAD

**Issue by minad** [minad/tempel#39]
It would be kind of nice if capture templates somehow trigger Tempel. I have to figure out a way to do this.

**Comment by jdtsmith** [comment]
You mean have a capture template which includes one or more tempel templates?  Very interesting.

**Comment by minad** [comment]
Yes somehow like that. I didn't experiment with that yet, but I imagine that this could be useful.

**Comment by minad** [comment]
Close due to inactivity. I didn't come up with good ideas for this yet.

**Comment by idlip** [comment]
I'd like to know how others are using tempel with capture.

Tempel helps to have single source of truth for all templates, so I decided to convert all capture into tempel itself since it can universally be used.

after trying with normal `%(tempel-insert 'name)` i noticed it simply inserts and there is no interactivity.
To not complicate with advice or hook, I simply went ahead with inserting only template name at point where i want and add cursor focus 

```elisp
  (setq org-capture-templates
   `(("j" "journal"      plain (file+olp+datetree "journal.org")    "journal-day%?"    :empty-lines 1)
))
```

with cursor at `%?` i just `M-+` which completes the template.
